src/common/SPELLS/shadowlands/conduits/shared.ts

```diff
@@ -0,0 +1,24 @@
+import { SpellList } from 'common/SPELLS/Spell';
```

Comment:
This isnt a conduit, and there is already a file for this.

common/SPELLS/shadowlands/soulbinds.js